meetings/2024-03-13.md

```diff
@@ -0,0 +1,164 @@
+* VB: Work item sounds good.
+* SC: I'm for the idea of more visibility. Suggest to have more care on the language, e.g., "best practice" or "guideline". If BP make sure there's sufficient data that signals that. Fine as a work item (best practice and guidelines).
+* eP: The naming is not most important. Just copied from OAuth Security BP. We need to be clear of course not to compare with stuff like that. Nor particularly comparable now. Early draft. Aspirations to have something like this in the future. We can rename in the future.
+* SC: Title is not the issue, but the language that's used in the document. Can discuss later.
+* RG: Let's make a work item and wait for it to reach a certain maturity to add to TR.
+* eP: As a rule of thumb: possible threats and countermeasures. If not conflicting with the spec, it is fair/valid approach. At least it is conforming for Solid group of specifications. If measure is passing the Solid test suites, it'd be useful to share it with others.
+
+#### Best Security Practice work item proposal
+URL: https://github.com/solid/specification/issues/636
+
+* VB: What are the next steps for this? Needs review?
+* eP: Where should it go? It'd be good to merge somewhere and then keep iterating. I'm the only one listed at the moment. I want to ask others to collaborate.
+* VB: Do that and we can discuss next week and in the meantime we can discuss where to put it.
+
+ACTION: eP to ask others to co-authors. The Group revisits the best place for the work to continue.
+
+* SC: Hard to say what best place is. Keep it in its own repo for now. Security practices cross all specs, not just Protocol. If we want to always work on it, we can think of it as a live document. I don't see it being the same as the Solid Protocol
+* eP: Also see it better as a dedicated repo for it. Easier to migrate into solid/specification.
+
+ACTION: VB to create repo with `foo-security-enterprises`.
+
+* SC: `solid-security-bp`?
+* VB: Will ask in chat.
+* eP: `security-topics`
+* SC: Nod.
+* SC: BP might be more familiarity in/around W3C.
+* eP: Follow up in issue.
+
+### Add Contributing Guide PR processing information
+URL: https://github.com/w3c-cg/solid/pull/19
+
+* VB: We already have w3c-cg/solid CONTRIBUTING.md for contributing all sorts of stuff. There is a section on meeting minutes recently. This PR is specifically about the Contributing Guide itself. I've approved it. It has been open for about a week? If you want to make a suggestion to the Contributing, this sets some expectations on processing.
+* eP: Right, if no objections, lets merge. I've approved.
+* VB: Merged.
+* SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We do't always hit deadlines or follow to the dot, but the more wwe take it seriously, the more we speed things up and the less chatter on how to work. Not about watching over each other's shoulder. 
```

Comment:
```suggestion
* SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We don't always hit deadlines or follow everything to the dot, but the more we succeed (or at least take it seriously), the more we speed things up and the less chatter on how to work. Not about watching over each other's shoulder. As mentioned before, it helps us to be more efficient and be on the same page, as well as being useful to newcomers and orientation.
```